CHANGELOG.md

```diff
@@ -9,7 +9,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
--
+- Users can now pause scan schedules by setting the `ScanSetting.suspend`
```

Comment:
This seems like a great QOL feature to add! I'll be sure it's highlighted in the release notes downstream in the future.